### PR TITLE
Update n-ui to v4

### PR DESCRIPTION
**Pull request by adambraimbridge**
This is *blocked*, pending a release of https://github.com/Financial-Times/n-email-article/tree/n-ui-v4.

The reason it's failing CircleCI is because you need to manually `bower install n-email-article#n-ui-v4`

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -24,6 +24,9 @@
   "resolutions": {
```
Comment (wheresrhys) [inline bower.json]: should not really need any resolutions. Try deleting them, then `rm -rf bower_components && bower install`

File: client/components/comments/main.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@import "n-ui/icons/main";
+@import "o-icons/main";
```
Comment (wheresrhys) [inline client/components/comments/main.scss]: not needed - n-ui already brings it in

File: client/main.scss [hunk 3]
```diff
@@ -1,16 +1,8 @@
-$n-ui-output-critical: true;
-
-@import 'n-ui/configure';
-@include nUiConfigure(
-	$features: (
-		buttons: true,
-		tourTip: true,
```
Comment (wheresrhys) [inline client/main.scss]: Looks like you probably need to `@import n-ui/tourtip/main`. maybe o-buttons/main too

**Comment by adambraimbridge** [comment]
Verbal 👍 from @wheresrhys. I'm running final tests now and plan to release before noon.

**Pull request merged**